src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -984,8 +984,6 @@ public function setCache(array $options)
      * @return $this
      *
      * @see http://tools.ietf.org/html/rfc2616#section-10.3.5
-     *
-     * @final
```

Comment:
Required because StreamedResponse overrides this method. Otherwise, we'll see `The "Symfony\Component\HttpFoundation\Response::setNotModified()" method is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Component\HttpFoundation\StreamedResponse".` as e.g. in https://travis-ci.org/symfony/symfony/jobs/426209061

Reply:
(to be discussed; there might be other solutions than this one, wdyt?)

Reply:
We could ignore `@final` annotations from the same namespace like we do for `@deprecated` but imo this is not worth it and maybe we even don't want this behavior